HW1-KZaballa.py

```diff
@@ -0,0 +1,181 @@
+# BDA 696 - Karenina Zaballa
+# Week 1 - HW1
+
+# installing libraries as needed
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+from sklearn.datasets import make_classification
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import train_test_split
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import Normalizer
+from sklearn.svm import SVC
+
+# Read in data files
+iris_df = pd.read_csv("D:/SDSU BDA/BDA 696/HW1/HW1-Data/iris.data.csv")

```

Comment:
Not the correct way to load the dataset since it'll use the first row as the header. Also, I obviously don't have this folder on my computer, so the code didn't run when I tried it.